test/config.test.js

```diff
@@ -620,144 +618,105 @@ test('valid serviceName => active', function (t) {
   t.end()
 })
 
-test('serviceName defaults to package name', function (t) {
```

Comment:
REVIEW NOTE: This suite of test cases was replaced by the 'serviceName/serviceVersion zero-conf: *' test cases below. Instead of having all this goop to create directory trees and packages in which to test, I just created the various test scenarios in "test/fixtures/pkg-zero-conf-*/" dirs.